compiler/modules/types/cell.bal

```diff
@@ -23,36 +23,26 @@ function cellSubtypeIsEmpty(Context cx, SubtypeData t) returns boolean {
 }
 
 function cellFormulaIsEmpty(Context cx, Conjunction? posList, Conjunction? negList) returns boolean {
-    SemType combined;
-    CellMutability minMut;
+    CellAtomicType combinedAtomicType;
     if posList == () {
-        combined = TOP;
-        minMut = CELL_MUT_UNLIMITED;
+        combinedAtomicType = { ty:TOP, mut:CELL_MUT_UNLIMITED };
```

Comment:
I think we call this something like `intersected` with other basic types.

Reply:
Used `combined` similar to other places.